conans/client/graph/graph_binaries.py

```diff
@@ -1,14 +1,15 @@
 import os
 
 from conans.client.graph.graph import (BINARY_BUILD, BINARY_CACHE, BINARY_DOWNLOAD, BINARY_MISSING,
-                                       BINARY_SKIP, BINARY_UPDATE, BINARY_WORKSPACE)
+                                       BINARY_SKIP, BINARY_UPDATE, BINARY_WORKSPACE,
+                                       RECIPE_EDITABLE, BINARY_EDITABLE)
 from conans.client.output import ScopedOutput
 from conans.errors import NoRemoteAvailable, NotFoundException
 from conans.model.info import ConanInfo
 from conans.model.manifest import FileTreeManifest
 from conans.model.ref import PackageReference
 from conans.util.env_reader import get_env
-from conans.util.files import is_dirty, rmdir
+from conans.util.files import rmdir, is_dirty
```

Comment:
I think that Pycharm is not ordering the list of imported things... it happens in many places (some lines above: ```from conans.errors import NoRemoteAvailable, NotFoundException```). I change it, but it will happen again in the future and in many places.